server/util/validators.js

```diff
@@ -4,7 +4,6 @@ import mongoConnection from '../util/mongo';
 mongoConnection();
 
 /** Checks if the email and password are valid
```

Comment:
TS doesn't complain at you for removing these? It usually does for me at work but I've been thinking about removing the requirement for the extra line... it's handy for dyslexia but kind of a waste of space.